oas_docs/output/kibana.yaml

```
@@ -51523,6 +51528,966 @@ paths:
+                - $ref: '#/components/schemas/Synthetics_tcpMonitorFields'
+        required: true
+      responses:
+        '200':
+          description: A successful response.
+      summary: Create a monitor
+      tags:
+        - synthetics
+  /api/synthetics/monitors/_bulk_delete:
+    post:
+      description: |
+        Delete multiple monitors by sending a list of config IDs.
+      operationId: delete-synthetic-monitors
+      requestBody:
+        content:
+          application/json:
+            examples:
+              bulkDeleteRequestExample1:
+                description: Run `POST /api/synthetics/monitors/_bulk_delete` to delete a list of monitors.
+                value: |-
+                  {
+                    "ids": [
+                      "monitor1-id",
+                      "monitor2-id"
+                    ]
+                  }
+            schema:
+              type: object
+              properties:
+                ids:
+                  description: An array of monitor IDs to delete.
+                  items:
+                    type: string
+                  type: array
+              required:
+                - ids
+        required: true
+      responses:
+        '200':
+          content:
+            application/json:
+              examples:
+                deleteMonitorsResponseExample1:
+                  description: A response from successfully deleting multiple monitors.
+                  value: |-
+                    [
+                      {
+                        "id": "monitor1-id",
+                        "deleted": true
+                      },
+                      {
+                        "id": "monitor2-id",
+                        "deleted": true
+                      }
+                    ]
+              schema:
+                items:
+                  description: The API response includes information about the deleted monitors.
+                  type: object
+                  properties:
+                    deleted:
+                      description: |
+                        If it is `true`, the monitor was successfully deleted If it is `false`, the monitor was not deleted.
+                      type: boolean
+                    ids:
+                      description: The unique identifier of the deleted monitor.
+                      type: string
+                type: array
+      summary: Delete monitors
+      tags:
+        - synthetics
+  /api/synthetics/monitors/{config_id}:
+    delete:
+      description: |
+        Delete a monitor from the Synthetics app.
+        You must have `all` privileges for the Synthetics feature in the Observability section of the Kibana feature privileges.
+      operationId: delete-synthetic-monitor
+      parameters:
+        - description: The identifier for the monitor that you want to delete.
+          in: path
+          name: config_id
```

```suggestion
          name: id
```

Same as noted before.

---

Fixed in ad6098364bd97d657d725f4aa3a2887fa7d1a9e4